pulsar-broker/src/main/java/org/apache/pulsar/broker/delayed/DelayedDeliveryTracker.java

```diff
@@ -31,6 +31,8 @@
 public interface DelayedDeliveryTracker extends AutoCloseable {
 
     /**
+     * may consider mark thread safe
```

Comment:
we can't mark for an interface, I don't think this comment make sense here as we also can't guarantee implementation is thread safe